### DISCUSSION TITLE
Android Webview Input type file does not show option dialog

### DISCUSSION THREAD

**Discussion by Spinnenzunge** [ionic-team/capacitor#3938] [Q&A]
Hi there.
I have an `<input type="file">` in a Capacitor Android Build and would expect on Select to see a Choice between Camera and File Browser (like I get on iOS or in the Chrome browser on Android). But instead it jumps directly to the File browser and does not let me choose between the available options. Do I miss something in my AndroidManifest or is this a Android Webview problem?

**Comment by jcesarmobile** [comment]
It’s like that by design, to use the camera you can use the capture attribute, but it’s not possible to prompt

**Reply by Spinnenzunge** [reply]
What is the reason behind that?
